.ci_support/migrations/python312.yaml

```diff
@@ -29,9 +30,9 @@ __migrator:
     exclude_pinned_pkgs: false
 
 python:
-  - 3.11.* *_cpython
+  - 3.12.* *_cpython
 # additional entries to add for zip_keys
 numpy:
-  - 1.23
+  - 1.26
```

Comment:
Prody cannot support such high python versions yet. We are still working towards that (see https://github.com/prody/ProDy/pull/1399 and https://github.com/prody/ProDy/pull/1655)